rclpy/src/rclpy/_rclpy_pybind11.cpp

```diff
@@ -226,4 +227,33 @@ PYBIND11_MODULE(_rclpy_pybind11, m) {
   m.def(
     "rclpy_wait", &rclpy::wait,
     "rclpy_wait.");
+
+  m.def(
+    "rclpy_get_topic_names_and_types",
+    &rclpy::graph_get_topic_names_and_types,
+    "Get all topic names and types in the ROS graph.");
+  m.def(
+    "rclpy_get_publisher_names_and_types_by_node",
+    &rclpy::graph_get_publisher_names_and_types_by_node,
+    "Get topic names and types for which a given remote node has publishers.");
+  m.def(
+    "rclpy_get_subscriber_names_and_types_by_node",
+    &rclpy::graph_get_subscriber_names_and_types_by_node,
+    "Get topic names and types for which a given remote node has subscribers.");
+  m.def(
+    "rclpy_get_service_names_and_types",
+    &rclpy::graph_get_service_names_and_types,
+    "Get all service names and types in the ROS graph.");
+  m.def(
+    "rclpy_get_service_names_and_types",
```

Comment:
It looks like the `rclpy_get_service_names_and_types` `m.def(` call is duplicated.

Reply:
Yeap, my bad. I wonder why `pybind11` didn't complain though. See https://github.com/ros2/rclpy/pull/717/commits/a13f583864c6a3ff3d68fbd577acb3e86361eb07.